src/test/java/amazin/service/BookServiceTest.java

```diff
@@ -51,6 +51,7 @@ public void saveTest() {
         verify(repository).save(book);
     }
 
+    @Test
```

Comment:
@zaidoon1 
This should have been caught when createTest was created; though 👍  @abungay for finding it